src/sql/query_util.py

```diff
@@ -27,3 +29,104 @@ def extract_tables_from_query(query):
         # fixtures. (#546). This function can also be moved to util.py
         # after #545 is resolved.
         return []
+
+
+def parse_sql_snippet(snippet):
```

Comment:
there are no unit tests for any of these functions

- parse_sql_snippet
- extract_metadata
- extract_main_sql
- wtrite_sql_with_markers
- _generate_dependencies_parser

Reply:
added these tests in a new file test_query_util.py